README.md

```diff
@@ -1,105 +1,256 @@
-```
+- Access to a k8s cluster
+- [Go](https://go.dev/)
+- [K3D](https://k3d.io/)
+- [docker](https://www.docker.com/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [kubebuilder](https://book.kubebuilder.io/)
 
-2. Build and push your image to the location specified by `IMG`:
-	
-```sh
-make docker-build docker-push IMG=<some-registry>/serverless-manager:tag
-```
-	
-3. Deploy the controller to the cluster with the image specified by `IMG`:
+## Installation on K3S cluster
 
-```sh
-make deploy IMG=<some-registry>/serverless-manager:tag
-```
+1. Clone project
 
-### Uninstall CRDs
-To delete the CRDs from the cluster:
+    ```bash
+    git clone https://github.com/kyma-project/serverless-manager.git && cd serverless-manager/
+    ```
 
-```sh
-make uninstall
-```
+2. Build manager locally and run in on k3s cluster
 
-### Undeploy controller
-UnDeploy the controller to the cluster:
+    ```bash
+    make k3d-run
+    ```
 
-```sh
-make undeploy
-```
+> Note: to creanup k3d cluster use the `make k3d-stop` make target.
 
-## Contributing
-// TODO(user): Add detailed information on how you would like others to contribute to this project
+## Manual installation of K3S cluster
 
-### How it works
-This project aims to follow the Kubernetes [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
+1. Clone project
 
-It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/) 
-which provides a reconcile function responsible for synchronizing resources untile the desired state is reached on the cluster 
+    ```bash
+    git clone https://github.com/kyma-project/serverless-manager.git && cd serverless-manager/
+    ```
 
-Serverless Manager watches `Serverless` CR and reconciles serverless components.
-When a Serverless CR is discovered the manager uses its content as configuration for all the serverless components.
-When the content of Serverless CR is updated by the user, the manager reconciles the serverless components to make them up to date with the desired configuration.
+2. Provision k3d cluster
 
->Warning: Uninstalling serverless leaves no funtions behind!
+    ```bash
+    kyma provision k3d
+    ```
 
-When `Serverless` CR is deleted the serverless manager undeploys all serverless workloads, uninstalls the `function.serverless.kyma-project.io` CRD and, as a consequence, deletes all function resources.
-It is important to realize that when you decide to uninstall serverless you will loose all the code stored inline in you functions.  
-Using git sourced functions and version-controlled application definition and configuration is recomended.
+3. Install prerequisites
 
+    ```bash
+    kyma deploy -s main --component cluster-essentials --profile production --ci
+    ```
 
-### Test It Out
-1. Install the CRDs into the cluster:
+    > Note: this step is required only because the serverless-manager is in early stage and there is no manager for kyma CRDs installation
```

Comment:
```suggestion
    > Note: this step is required only because `serverless-manager` is in the early stage, and there is no manager for the Kyma CRDs installation.
```